CONTRIBUTING.md

```diff
@@ -206,14 +206,16 @@ when making and submitting changes.
 
 All modifications to non-Paper files should be marked.
 
-- Multi-line changes start with `// Paper start` and end with `// Paper end`;
+- Multi-line changes start with `// Paper start - <COMMIT DESCRIPTION>` and end
+  with `// Paper end - <COMMIT DESCRIPTION>`;
 - You can put a comment with an explanation if it isn't obvious, like this:
```

Comment:
Should this wording be updated to make it sound required? I think its left over from when these were optional.